node/flatpak-node-generator.py

```diff
@@ -960,6 +960,38 @@ async def _handle_playwright(self, package: Package) -> None:
             self.gen.add_data_source("flatpak-node-cache",
                                      destination=destdir / 'INSTALLATION_COMPLETE')
 
+    async def _handle_esbuild(self, package: Package) -> None:
+        pkg_names = {
+            'x86_64': 'esbuild-linux-64',
+            'i386': 'esbuild-linux-32',
+            'arm': 'esbuild-linux-arm',
+            'aarch64': 'esbuild-linux-arm64'
+        }
+
+        for flatpak_arch, pkg_name in pkg_names.items():
+            dl_url = f'https://registry.npmjs.org/{pkg_name}/-/{pkg_name}-{package.version}.tgz'
+            metadata = await RemoteUrlMetadata.get(dl_url, cachable=True)
+
+            archive_dst = self.gen.data_root / 'cache' / 'esbuild' / '.package' / f'{pkg_name}@{package.version}'
```

Comment:
(very delayed, sorry!) nit, but couldn't this be `cache_dst / '.package' / ...` if the order is flipped?

Reply:
Not sure if I get this. What's the difference? Should I define `cache_dst = self.gen.data_root / 'cache' / 'esbuild'`?

Reply:
Done.